admin/i18n/de/translations.json

```
@@ -22,6 +23,7 @@
     "Product": "Produkt",
     "Remove node": "Gerät entfernen",
     "Save Config:": "Konfiguration speichern:",
+    "Select group": "Gruppe wählen",
```

Um konsistent zu bleiben:
```suggestion
    "Select group": "Gruppe auswählen",
```

words.js nicht vergessen!

---

done